### ISSUE TITLE
How to add tf.keras.layers.AdditiveAttention in my model?

### ISSUE THREAD

**Issue by thisisdhruvagarwal** [keras-team/keras#14246]
I am opening an issue for the first time, so please forgive if I done any mistakes. 

I am working on a machine language translation problem. The Model I am using is:

    Model = Sequential([
          Embedding(english_vocab_size, 256, input_length=english_max_len, mask_zero=True),
          LSTM(256, activation='relu'),
          RepeatVector(german_max_len),
          LSTM(256, activation='relu', return_sequences=True),
          Dense(german_vocab_size, activation='softmax')
    ])

Here, english_vocab_size and english_max_len are the total number of english words in the english vocabulory and number of words in each english sentence respectively. And the same is with german_vocab_size and german_max_len.

Now, I wanted to make this model more better by adding Bahdanau Attention in it. So, How can I add tf.keras.layers.AdditiveAttention layer, which is the same as Bahdanau Attention, in this Model?

I tried to find good tutorials on google for the implementation of tf.keras.layers.AdditiveAttention layer on an nlp task, but couldn't find any.

I asked this question on various different sites like stackoverflow, Keras Users Google Group, Keras Slack Channel, Keras on Gitter, and on Quora, but didn't received any answers till now. Here are the links for all my questions asked on various sites: [On StackOverflow](https://stackoverflow.com/questions/64301624/how-can-i-add-tf-keras-layers-additiveattention-in-my-model), [Keras Users Google Group](https://groups.google.com/g/keras-users/c/oiHSlPM_pkQ/m/h22dz8LwAwAJ), [Keras Slack Channel](https://kerasteam.slack.com/archives/C5974N9E2/p1602415913001400), [Keras On Gitter](https://gitter.im/Keras-io/Lobby#) and [Quora](https://qr.ae/pNM0RR)

**Comment by thisisdhruvagarwal** [comment]
Sorry, mouseslip
